frontend/packages/console-shared/src/components/status/SecondaryStatus.tsx

```diff
@@ -10,7 +10,7 @@ const SecondaryStatus: React.FC<SecondaryStatusProps> = ({ status }) => {
   if (statusLabel) {
     return (
       <div>
-        <small className="text-muted">{statusLabel}</small>
+        <small className="co-status-card__health-item-subtitle text-muted">{statusLabel}</small>
```

Comment:
Per BEM, this class should only be applied in the health card. I believe `SecondaryStatus` is used in other contexts.

Is it possible to take in a className prop?

Reply:
http://getbem.com/introduction/

Reply:
Modified the code to use `SecondaryStatus` component to render subtitle for `Health Checks`